### DISCUSSION TITLE
Ocarina Of Time, Error Object Eye or Lens of Truth

### DISCUSSION THREAD

**Discussion by JugelsitoYT** [mupen64plus-ae/mupen64plus-ae#913] [General]
I recently got an error while playing Ocarina Of Time, it turns out that when using the object of the Lens or Eye of Truth, when uploading or moving objects that it reveals, the game quits and sends me directly to the main menu of the emulator It also happens when selecting enemies with Z, I hope they solve this error just like in Majora's Mask, thank you very much. 

PostData: 'IF I USE THE ORIGINAL PLAY STORE APP'. Thanks...

**Comment by fzurita** [comment]
What device do you have?

What area is it crashing on with the lens of truth?

Are you using any ROM hacks or cheats?

**Reply by JugelsitoYT** [reply]
Hello good Day. 

I use a Moto G5 device, Android version 8.1.0. 

The bug is introduced at the beginning of the Shadow Dungeon when moving the first statue towards the real skull. 

I don't use tricks, I play a virgin.

**Reply by fzurita** [reply]
Interesting, I thought I fixed all the crashes on that game when Glide64.

Try this, select the game, go to settings, change the emulation profile to GLideN64-Fast or GLideN64-Medium if your device can handle it.

**Reply by JugelsitoYT** [reply]
Ok, solve the error pot now by putting it in Gln64-Accurate, thank you very much, the other options you gave me happened the same, I will be informing in the forum, if I find any other error, thank you very much, I also hope that you investigate that error, many Thank you.
